docs/faq/lukso/project-origins.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: 'Project Origins'
+sidebar_position: 6
+---
+
+# Project Origins
+
+### What was the LUKSO rICO?
+
+The public funding of the project took place in 2020 on the Ethereum blockchain using Ether. LYXe, the **token equivalent on the Ethereum blockchain**, was handed out to participants of the funding.
+
+The public sale was implemented as a **Reversible Initial Coin Offering** (rICO), allowing participants to purchase LYXe tokens at a specific price over **eight months** without a commitment to participate in the entire process length of the sale. Buyers could discontinue and have their remaining ETH returned at any time. It was also possible to join the sale in the latter stages without being drastically disadvantaged. In total, the project received nearly 10,000 Ether.
+
+The rICO, proposed by LUKSO founder Fabian Vogelsteller for the LUKSO public sale, is a **public standard that gradually unlocks tokens through an autonomous-acting contract instance** deployed on the Ethereum blockchain. LUKSO did this to provide a fairer token sale and more protection for the buyer.
```

Comment:
maybe you can add links here to rICO related article, rICO smart contract, rICO website

https://rico.lukso.network/

Reply:
Wow, great news. I did not know that this page still exists!